__tests__/app.test.js

```diff
@@ -159,4 +138,49 @@ describe("GET /api/articles/:article_id", () => {
                 expect(body.msg).toBe("Bad Request")
             })
     })
-})
\ No newline at end of file
+})
+
+describe("GET /api/articles/:article_id/comments", () => {
+    test("response contains array of comments object with matching IDs and correct properties", () => {
+        return request(app)
+            .get("/api/articles/3/comments")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.comments).toBeInstanceOf(Array)
+                body.comments.forEach(comment => {
+                    expect(comment).toMatchObject({
+                        comment_id: expect.any(Number),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String),
+                        author: expect.any(String),
+                        body: expect.any(String),
+                        article_id: 3
+                    })
+                })
+            })
+        })
+        test("comments array is sorted by most recent first", () => {
```

Comment:
what if it is a valid article id but there are no comments associated with it 🧐 this would still give us a 200, can we add this test?